tests/testthat/indention_token_dependent/token_dependent_comments-out.R

```diff
@@ -0,0 +1,7 @@
+call(call( # comment
+  3, 4
+))
+
+call(call(1, # comment
+          3
```

Comment:
Ok. Then, I guess the question is whether we should break after `(` by default. Also, what means "by default"? I would suggest to make it dependent on the `strict` argument. Alternatively, we can define another level of invasiveness. 
See #118.